.infra/index.ts

```diff
@@ -522,3 +523,25 @@ if (vpcNativeProvider) {
     { provider: vpcNativeProvider.provider },
   );
 }
+
+deployClickhouseSync(
```

Comment:
This syncs all api tables into clickhouse?

Reply:
nope, only specified [here](https://github.com/dailydotdev/daily-api/pull/1835/files#diff-7eb3c0dc4b6e8082a5405da027b746c2ae8567945714b1a8497351207852c6ceR28)

Reply:
We probably want to run this only in production since in local development it will fail due to missing clickhouse variables.

Reply:
we going to move it to pulumi-common eventually, and it will be configurable the same way we configure debezium.

Reply:
@omBratteng do you think it will take time to move it to pulumi common? if not maybe we can do it right away to prevent the migration later?

Reply:
Since we'll use this in other services later, we can move it to pulumi-common from the get-go

Reply:
@omBratteng would be great if you can take care of it please 🙏

Reply:
Will do it today

Reply:
Created new PR for it #1858